CHANGELOG.md

```diff
@@ -3,6 +3,344 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.9
+
+_Jan 18, 2024_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Allow to filter non-filterable columns programmatically (#11538) @MBilalShafi
```

Comment:
@romgrk I feel one like can't do justice to the improvement, a blogpost would feel much better, feel free to rephrase or add more information though
```suggestion
- 🎁 The Data Grid headers are now sticky - which brings immense improvements to scrolling, state management, and overall performance of the grid
- ⚙️ The Data Grid disabled column-specific features like (filtering, sorting, grouping, etc) could now be accessed programmatically. See the related [docs](https://next.mui.com/x/react-data-grid/api-object/#access-the-disabled-column-features) section.
```